### DISCUSSION TITLE
On/Off option for highlights and ignores

### DISCUSSION THREAD

**Discussion by Shrezno** [Chatterino/chatterino2#3616] [Ideas]
For busy chats I like to have myself on highlights but in slow chats its just in the way and annoying so I have to add and remove myself from highlights constantly. 

I think it would be a good addition to have an on/off option for both highlights and ignores.

Heres an artistic rendition of what I imagine it could look like: https://i.nuuls.com/pELHw.png

**Comment by LosFarmosCTL** [comment]
I agree with this, should be a thing. 
Especially when using complex highlight/ignore regex patterns, storing them in a separate location to be able to delete and re-add when you just want to turn off/on something for a limited time is pretty annoying.
